frontend/src/app/main/ui/messages.cljs

```diff
@@ -37,7 +37,12 @@
                            :bottom-actions (= controls :bottom-actions))
                    :data-test data-test
                    :role role}
-     content
+     [:span
+      content
+      (for [link links]
+        [:* {:key (uuid/next)}
```

Comment:
Don't set random key just for satisfy react, derive the key from link or use the `(:label link)` or index

Reply:
Done